pulp_puppet_plugins/pulp_puppet/plugins/migrations/0005_puppet_module_name_change.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+from mongoengine import NotUniqueError
+
+from pulp.server.controllers import repository as repo_controller
```

Comment:
This can be grouped with line L#3 since it's an external module but not part of the standard library (similar to `mongoengine`)